libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts

```diff
@@ -33,9 +36,15 @@ export const ActionSquashBreakpointPx = 1280;
     encapsulation: ViewEncapsulation.None,
     host: {
         '[attr.tabindex]': '0'
-    }
+    },
+    providers: [
+        {
+            provide: DYNAMIC_PAGE_HEADER_TOKEN,
+            useExisting: forwardRef(() => DynamicPageHeaderComponent)
```

Comment:
just FYI, forwardRef is not required when referring the component directly in the director